src/bbs2gh/Commands/GenerateScript/GenerateScriptCommandArgs.cs

```diff
@@ -27,6 +27,7 @@ public class GenerateScriptCommandArgs : CommandArgs
     public bool KeepArchive { get; set; }
     public bool NoSslVerify { get; set; }
     public string TargetApiUrl { get; set; }
+    public bool UseGithubStorage { get; set; }
```

Comment:
We need to add a validation (and an extra test) for when both `AwsBucketName` and `UseGithubStorage` are provided.